doc-Installing_on_OpenShift_Container_Platform/topics/installation.adoc

```diff
@@ -26,7 +26,18 @@ To successfully deploy a {product-title} appliance on OpenShift Container Platfo
 OpenShift Container Platform 3.5 is required for this installation. Red Hat has not tested this procedure with earlier versions of OpenShift Container Platform.
 ====
 
-The {product-title_short} deployment uses three files to create the appliance: `cfme-template.yaml`, which is the {product-title_short} template used for the deployment, and `cfme-pv-example.yaml` and `cfme-pv-app-example.yaml`, two pod volume files. OpenShift Container Platform 3.5 includes these files by default.
+The {product-title_short} deployment uses three files to create the appliance: `cfme-template.yaml`, which is the {product-title_short} template used for the deployment, and `cfme-pv-example.yaml` and `cfme-pv-app-example.yaml`, two pod volume files. 
+
+These files create the OpenShift pods comprising the appliance, and are available from the Red Hat Container Catalog. To download the files to your system, follow the OpenShift instructions on the *Get this image* page for each of the following images:
```

Comment:
@dayleparker The updates look great and the instructions are easy to understand. Though I was just wondering if we should make this step a bit more explicit like in a procedure, something along the lines of  (sorry to nitpick!) -
"To download the files to your system, click the *Get this image* tab? and select *OpenShift* from the *Choose your platform:* list, then follow the instructions on the page for each image:"  
What do you think?